ibis/sql/alchemy.py

```diff
@@ -509,6 +509,75 @@ def build_ast(expr, context=None, dialect=None):
     return builder.get_result()
 
 
+class AlchemyDatabaseSchema(Database):
+
+    def __init__(self, name, database):
+        """
+
+        Parameters
+        ----------
+        name : str
+        database : AlchemyDatabase
+        """
+        self.name = name
+        self.database = database
+        self.client = database.client
+
+    def __repr__(self):
+        return "{0}('{1}')".format('Schema', self.name)
+
+    def drop(self, force=False):
+        """
+        Drop the schema
+
+        Parameters
+        ----------
+        force : boolean, default False
+          Drop any objects if they exist, and do not fail if the database does
+          not exist.
+        """
+        raise NotImplementedError()
```

Comment:
Can you give this exception a message to let users know what exactly isn't implemented? Something like "drop is not implemented for postgres schemas" or something like that.